R/panel.R

```diff
@@ -0,0 +1,21 @@
+#' Creates a panel for all info boxes so they do not overlap
+#'
+#' @param ... calls with info elements
+#' @param position character with position of the parameter. Default "top right".
+#'
+#' @return div which wraps your all info boxes to display it in the position corner of your shiny app.
+#' @export
+info_panel <- function(..., position = "top right") {
+  display(tagList(...), position = position, type = "info_panel") #nolint
```

Comment:
tagList should be imported here

Reply:
and then probably you can lint

Reply:
> I run it with just sys.call() all works just fine.
Yeap, there used to be just `sys.call(-8)` and was working perfectly when I've run example app, but failed for `devtools::check()`. And that was the reason why CI through error as -8 was outside the call stack. 
If you had in mind something that solves the CI issue please explain.

Reply:
Why is `-8` even needed? I suggested using `sys.call()` on its own. It passed my checks locally.

BTW I have no idea why this comment landed here.

Reply:
Maybe using simple `sys.call()` pass the checks, but the code won't work as it should be. It returns the `display` function always. The whole idea here is to know whether current info box element was called from within `info_panel` or not. If so then the `position` argument must be overwritten to that used in `info_panel` and css `position`needs be removed (so that elements do not overlap).
I've modified it to nicer and safer solution with `tryCatch`